### PR TITLE
Update 'how it works' for Password input

### PR DESCRIPTION
**Pull request by edbrissenden**
Made a clearer description of how to write labels and toggles when showing one or more password inputs.

### CODE REVIEW

**Review by owenatgov — APPROVED** [review]
Looks good to me! Just needs a second approval.

File: src/components/password-input/index.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ Hide passwords by default until the user chooses to show it using the ‘show’
 
 If you do choose to include two or more password inputs on a page, the ‘show’ and ‘hide’ toggles and labels for each password input must be different.
 
-For example, you can label the input “Password” as “show password” and label the second input “Re-enter password” as “show re-entered password”.
+For example, you can label the first input as “Password” with a toggle of “show password” and label the second input “Re-enter password” with a toggle of “show re-entered password”.
```
Comment (calvin-lau-sig7) [inline src/components/password-input/index.md:89]:
```suggestion
For example, you can label:
- the first input as 'Password' with a toggle of 'show password'
- the second input as 'Re-enter password' with a toggle of 'show re-entered password'
```

**Comment by calvin-lau-sig7** [comment]
Thanks for this contribution @edbrissenden! 
Looks good to me too, I've also committed an edit to present as a bullet list to read a bit better.

**Review by calvin-lau-sig7 — APPROVED** [review]

**Comment by owenatgov** [comment]
@calvin-lau-sig7 It looks like your change upset prettier 😬 Would you be ok to run this (`npm run lint:prettier -- --write`) or do you want me to do it?

**Comment by calvin-lau-sig7** [comment]
I can't do command lines, but I'll fix it manually, I'm assuming it's my bullet points.

File: src/components/password-input/index.md [hunk 2]
```diff
@@ -86,7 +86,9 @@ Hide passwords by default until the user chooses to show it using the ‘show’
 
 If you do choose to include two or more password inputs on a page, the ‘show’ and ‘hide’ toggles and labels for each password input must be different.
 
-For example, you can label the input “Password” as “show password” and label the second input “Re-enter password” as “show re-entered password”.
+For example, you can label:
+- the first input as 'Password' with a toggle of 'show password'
+- the second input as 'Re-enter password' with a toggle of 'show re-entered password'
```
Comment (calvin-lau-sig7) [inline src/components/password-input/index.md:91]:
```suggestion
For example, you can label:

- the first input as 'Password' with a toggle of 'show password'
- the second input as 'Re-enter password' with a toggle of 'show re-entered password'
```

**Pull request merged**